### ISSUE TITLE
Windows Client Bandwidth Limit

### ISSUE THREAD

**Issue by 1karu5** [haiwen/seafile-client#755]
The upload speed limit is ignored in the windows client. I am using Windows 10 and client version 5.0.0 or 4.4.2.

**Comment by shoeper** [comment]
I could imagine that this has been introduced while adding multi threaded downloading.

**Comment by 1karu5** [comment]
This is with client 5.0.0, previous version (i think it was 4.4.2) had the same issue. Are my settings wrong?

![seafile](https://cloud.githubusercontent.com/assets/4019270/11609335/7c0a36a0-9b84-11e5-9e31-ddeb4bf703be.PNG)

**Comment by shoeper** [comment]
It looks like the client downloads 30 kiB/s too fast. Maybe only data but no metadata is counted.

**Comment by 1karu5** [comment]
maybe, but 80/90 kb/s could be my maximum upload speed. but as i see, i am able to limit it...but still to the wrong value

![seafile2](https://cloud.githubusercontent.com/assets/4019270/11609414/9acfcc56-9b86-11e5-98d9-ce363ca05f87.PNG)

**Comment by silberzwiebel** [comment]
I do have a similar problem with seafile-client 5.1.1 under Fedora Linux. I've set download limit to 50 kB/s, but the system monitor shows very clearly a download speed around 124 kB/s.

**Comment by killing** [comment]
The limit may be not so accurate due to our implementation. We'll consider improve it when time permits.

**Comment by hampelratte** [comment]
Yeah, this is very inaccurate. You have to play around with the value and monitor the used bandwidth to get the desired result. I wanted to restrict it to 70 KiB/s and had to set it to 40 KiB/s.
